plugins/modules/dcnm_res_manager.py

```diff
@@ -0,0 +1,1254 @@
+        scope_type: "fabric"                    # choose from ['fabric', 'device', device_interface', 'device_pair', 'link']
+        resource: "101"                         # The value of the resource being created
+
+      - entity_name: "9M99N34RDED~9NXHSNTEO6C"  # A unique name to identify the resource
+        pool_type: "ID"                         # choose from ['ID', 'IP, 'SUBNET']
+        pool_name: "VPC_ID"                     # Based on the 'poolType', select appropriate name
+        scope_type: "device_pair"               # choose from ['fabric', 'device', device_interface', 'device_pair', 'link']
+        switch:                                 # provide the switch information to which the given resource is to be attached
+          - 192.175.1.1
+          - 192.175.1.2
+        resource: "500"                         # The value of the resource being created
+
+      - entity_name: "mmudigon-2"               # A unique name to identify the resource
+        pool_type: "IP"                         # choose from ['ID', 'IP, 'SUBNET']
+        pool_name: "LOOPBACK0_IP_POOL"          # Based on the 'poolType', select appropriate name
+        scope_type: "fabric"                    # choose from ['fabric', 'device', device_interface', 'device_pair', 'link']
+        resource: "110.1.1.1"                   # The value of the resource being created
+
+      - entity_name: "9M99N34RDED~Ethernet1/10" # A unique name to identify the resource
+        pool_type: "IP"                         # choose from ['ID', 'IP, 'SUBNET']
+        pool_name: "LOOPBACK1_IP_POOL"          # Based on the 'poolType', select appropriate name
+        scope_type: "device_interface"          # choose from ['fabric', 'device', device_interface', 'device_pair', 'link']
+        switch:                                 # provide the switch information to which the given resource is to be attached
+          - 192.175.1.1
+        resource: "fe:80::04"                   # The value of the resource being created
+
+      - entity_name: "9M99N34RDED~Ethernet1/3~9NXHSNTEO6C~Ethernet1/3"  # A unique name to identify the resource
+        pool_type: "SUBNET"                     # choose from ['ID', 'IP, 'SUBNET']
+        pool_name: "SUBNET"                     # Based on the 'poolType', select appropriate name
+        scope_type: "link"                      # choose from ['fabric', 'device', device_interface', 'device_pair', 'link']
+        switch:                                 # provide the switch information to which the given resource is to be attached
+          - 192.175.1.1
+        resource: "fe:80:05::05/64"
+
+# DELETING RESOURCES
+# ==================
+
+- name: Delete Resources
+  cisco.dcnm.dcnm_res_manager:
+    state: deleted                              # choose form [merged, deleted, query]
+    fabric: test_fabric
+    config:
+      - entity_name: "l3_vni_fabric"            # A unique name to identify the resource
```

Comment:
Can we delete the resource using just the `entity_name`?

Reply:
No. All parameters are mandatory